Makefile

```diff
@@ -35,6 +35,7 @@ stop:
 run-dev:
 	docker build src/. -t pyroapi:python3.8-alpine3.10
 	docker-compose -f docker-compose-dev.yml up -d --build
+    awslocal s3api create-bucket --bucket sample-bucket
```

Comment:
slight problem of indentation I think no?

Reply:
weird, I can't seem able to fix it
<img width="457" alt="image" src="https://user-images.githubusercontent.com/33450491/228006055-0572f50f-ae2f-42e6-b50c-e84d56e91785.png">